src/components/fair/booth.tsx

```diff
@@ -0,0 +1,179 @@
+import * as React from "react"
+import * as Relay from "react-relay"
+import styled from "styled-components"
+
+import Artworks from "../artwork_grid"
+import Button from "../buttons/ghost"
+import FollowButton from "../follow"
+import Text from "../text"
+
+import * as fonts from "../../assets/fonts"
+
+const PageSize = 8
+
+interface Props extends RelayProps, React.HTMLProps<FairBooth> {
+  show: any
+  relay?: any,
+  onContactGallery?: (showId: number) => any,
+}
+
+interface State {
+  loading: boolean,
+}
+
+export class FairBooth extends React.Component<Props, State> {
+
+  constructor(props) {
+    super(props)
+    this.state = {
+      loading: false,
+    }
+  }
+
+  loadMoreArtworks() {
+    const hasMore = this.props.show.artworks.pageInfo.hasNextPage
+    if (!this.state.loading && hasMore) {
+      this.setState({ loading: true }, () => {
+        this.props.relay.setVariables({
+          artworksSize: this.props.relay.variables.artworksSize + PageSize,
+        }, readyState => {
+          if (readyState.done) {
+            this.setState({ loading: false })
+          }
+        })
+      })
+    }
+  }
+
+  render() {
+    const { show, onContactGallery, relay } = this.props
+
+    const showLocation = show.location && show.location.display && (
+      <LocationText textSize="small" textStyle="secondary">
+        {show.location.display}
+      </LocationText>
+    )
+
+    const artworksLeft =  show.counts.artworks - relay.variables.artworksSize
+
+    const loadMoreButton = (
+      <LoadMoreContainer>
+        <LoadMoreButton onClick={() => this.loadMoreArtworks()}>
+          See {artworksLeft} more artworks
+        </LoadMoreButton>
+      </LoadMoreContainer>
+    )
+
+    return (
+      <div>
+        <Header>
+          <div>
+            <PartnerLine>
+              <PartnerText textSize="small" textStyle="primary">
+                {show.partner.name}
+              </PartnerText>
+              <FollowButton type="profile" profile={show.partner.profile}/>
+            </PartnerLine>
+            {showLocation}
+          </div>
+          <div>
+            <Button onClick={() => onContactGallery(show.id)}>
```

Comment:
You’re passing a string (`show.id`) here to the callback, however [the callback’s signature](https://github.com/artsy/reaction-force/pull/175/files#diff-12ccaa558d908f4178256ea8b8b75bacR17) is defined to take a `number`.

This was probably not raised by the compiler because your `Props` interface overrides the `show` field typings, so that answers [this question](https://github.com/artsy/reaction-force/pull/175/files#r123081031), I guess.